zipkin-lens/src/translations/es.json

```diff
@@ -0,0 +1,32 @@
+{
+  "src.components.App.changeLanguage": "Cambiar Idioma",
+  "src.components.DiscoverPage.TracesTab.duration": "Duración",
+  "src.components.DiscoverPage.TracesTab.filter": "Filtro",
+  "src.components.DiscoverPage.TracesTab.numResults": "{num, number} {num, plural,\n    one {Result}\n    other {Results}\n  }",
+  "src.components.DiscoverPage.TracesTab.root": "Raíz",
+  "src.components.DiscoverPage.TracesTab.serviceName": "Nombre del Servicio",
+  "src.components.DiscoverPage.TracesTab.startTime": "Tiempo de Inicio",
+  "src.components.DiscoverPage.TracesTab.traceId": "ID de Traza",
+  "src.components.DiscoverPage.discover": "Descubrir",
+  "src.components.DiscoverPage.searchTracesDescription": "Por favor, selecciona un criterio en la barra de búsqueda. Luego, haz clic en el botón buscar.",
+  "src.components.DiscoverPage.searchTracesHeader": "Buscar Trazas",
+  "src.components.GlobalSearch.limit": "Límite",
+  "src.components.GlobalSearch.pleaseSelectCriteria": "Por favor, selecciona el criterio para tu búsqueda de trazas",
+  "src.components.TracePage.depth": "Profundidad",
```

Comment:
Not sure about this. While "profundidad" is the literal translation I would think of "niveles" here. If you have a screenshot maybe @eleduardo can help?